src/Cake.DotNetTool.Module/DotNetToolPackageInstaller.cs

```diff
@@ -207,7 +215,8 @@ private List<DotNetToolPackage> GetInstalledTools(string toolLocation)
                     {
                         Id = match.Groups["packageName"].Value,
                         Version = match.Groups["packageVersion"].Value,
-                        ShortCode = match.Groups["packageShortCode"].Value
+                        ShortCode = match.Groups["packageShortCode"].Value,
+                        Manifest = match.Groups["packageManifest"].Value

```

Comment:
I've done no testing so far, but expecting this to be empty and not throw when not matched.